README.md

```diff
@@ -5,14 +5,15 @@ Sfi.ru Distribution
 
 ###Install with Docker
 
-1. Install Docker
+1. Install Docker latest version. Guide for Ubuntu is here [Docker-maintained Package Installation](https://docs.docker.com/installation/ubuntulinux/#ubuntu-trusty-1404-lts-64-bit)
 2. Create a directory where your project files will be. By convention `/home/username/docker/project-name`
-3. Copy contents of https://github.com/sfi-ru/SfiDistr/blob/master/docker/docker-compose.yml and replace `PATH_TO_PROJECT_FILES` with smth like `/home/username/docker/`
-4. For local development:
-  * Add local domain name (e.g. `sfi` instead of `sfi.ru`) to `/etc/hosts` and point it to `127.0.0.1`
-  * Adjust `T3APP_NAME` in `docker-compose.yml` with local domain name
+3. Add local domain name (e.g. `sfi` instead of `sfi.ru`) to `/etc/hosts` and point it to `127.0.0.1`
```

Comment:
The whole point of this guide is that it should cover both development and production setup, that's why I wanted to group everything related to local dev setup, because you would not need to do those steps in production.